### ISSUE TITLE
ProfileNotFound: Profile Not Found

### ISSUE THREAD

**Issue by dedarh** [javilobo8/pubg-api-redis#8]
api.getAccountBySteamID('76561198044644802').then((account) => {					
					console.log(account.Nickname);
                    
                });

error  Unhandled rejection ProfileNotFound: Profile Not Found  but https://pubgtracker.com/api/search?steamId=76561198044644802 work 

    at ProfileNotFound (D:\1\pubg\pubg\node_modules\pubg-api-redis\src\pubg-erro
rs.js:22:5)
    at Promise.resolve.then.err (D:\1\pubg\pubg\node_modules\pubg-api-redis\src\
client.js:60:17)
    at tryCatcher (D:\1\pubg\pubg\node_modules\bluebird\js\release\util.js:16:23
)
    at Promise._settlePromiseFromHandler (D:\1\pubg\pubg\node_modules\bluebird\j
s\release\promise.js:512:31)
    at Promise._settlePromise (D:\1\pubg\pubg\node_modules\bluebird\js\release\p
romise.js:569:18)
    at Promise._settlePromise0 (D:\1\pubg\pubg\node_modules\bluebird\js\release\
promise.js:614:10)
    at Promise._settlePromises (D:\1\pubg\pubg\node_modules\bluebird\js\release\
promise.js:693:18)
    at Async._drainQueue (D:\1\pubg\pubg\node_modules\bluebird\js\release\async.
js:133:16)
    at Async._drainQueues (D:\1\pubg\pubg\node_modules\bluebird\js\release\async
.js:143:10)
    at Immediate.Async.drainQueues (D:\1\pubg\pubg\node_modules\bluebird\js\rele
ase\async.js:17:14)

**Comment by dedarh** [comment]
if 
api.getProfileByNickname('Paskuda73').then((profile) => {
                     console.log(profile);
});
Unhandled rejection StatusCodeError: 500 - "\n<!DOCTYPE html>\n<html>\n<head>\n<
meta name=\"viewport\" content=\"width=device-width\" />\n<title>Error</title>\n
</head>\n<body>\n<h2>\nSorry, an error occurred while processing your request.\n
</h2>\n</body>\n</html>"

**Comment by kuwagoo** [comment]
Pubgtracker API is currently disabled.
 [PubgTracker announcement](https://pubgtracker.com/article/14/status-on-our-discord-bot-twitch-bot-and-api)

**Comment by javilobo8** [comment]
Yes, Pubgtracker API is currently disabled... :( And my website is useless xD lobobot.com

**Comment by dedarh** [comment]
@javilobo8  can be rewritten library for parse html ?

**Comment by dedarh** [comment]
@javilobo8  in main page in js code playerData seasonInfo all statistics

`<script type="text/javascript">var seasonInfo = {"1":{"Id":1,"InternalName":"2017-pre1","Name":"Early Access Season #1","StartDate":"2017-03-27T00:00:00","EndDate":"2017-05-31T00:00:00","IsVisible":true},"2":{"Id":2,"InternalName":"2017-pre2","Name":"Early Access Season #2","StartDate":"2017-06-01T00:00:00","EndDate":"2017-07-31T00:00:00","IsVisible":true},"3":{"Id":3,"InternalName":"2017-pre3","Name":"Early Access Season #3","StartDate":"2017-08-01T00:00:00","EndDate":"2017-08-31T00:00:00","IsVisible":true},"4":{"Id":4,"InternalName":"2017-pre4","Name":"Early Access Season #4","StartDate":"2017-08-31T00:00:00","EndDate":null,"IsVisible":true}};</script>`

**Comment by dedarh** [comment]
`var request = require('request');
request('http://pubgtracker.com/pubg/search?steamId=76561198039584427', function (error, response, html) {
if (!error && response.statusCode == 200) {
var str = html;
var result = str.match( /playerData = .+}/i );
console.log("/br");
var resultat = JSON.parse(result[0].replace("playerData = "," "));
console.log(resultat.MatchHistory[0].Id);
}
});`

**Comment by dedarh** [comment]
@javilobo8  http://api.pubgtop.ru:81/api/search/76561198039584427

**Comment by Christian-nylander** [comment]
Is it still disabled? Im getting ProfileNotFound: Profile Not Found...

**Comment by javilobo8** [comment]
Yes, it still. https://imgur.com/a/JrhRO =(

**Comment by Christian-nylander** [comment]
Ohh :( But then what API is pubgtracker using? They  are getting data right?

**Comment by javilobo8** [comment]
I don't know how are they getting the data, there is a method (not sure if it is working nowadays) but you can check it out. https://github.com/Hormold/pubg

**Comment by Christian-nylander** [comment]
Oh alright! Has the API been down for a long time? Or is it something new? :)

**Comment by dedarh** [comment]
@Christian-nylander  I can help you with api

**Comment by javilobo8** [comment]
I don't know, I think they cannot maintain their api haha

**Comment by dedarh** [comment]
@javilobo8  https://pubg.op.gg  gives more information on the game and statistics and works more stable

**Comment by Christian-nylander** [comment]
@dedarh Oh nice! Where can i find the API they are using?

**Comment by javilobo8** [comment]
@dedarh Wow, did't know anything about that page. Great work from op.gg members. But there is no public api haha Maybe they are partnered with PUBG and have a private API.

**Comment by Christian-nylander** [comment]
What is this then? https://www.npmjs.com/package/pubg.op.gg

**Comment by dedarh** [comment]
@Christian-nylander  Contact me vk.me/dedarh OR Email: p.artem123@ya.ru  I'll show an example of how to look at the data from pubgtracker
@javilobo8 I do not know where they api, but maybe they use it  http://developers.overwolf.com/documentation/sdk/overwolf/games/events/pubg/ or https://github.com/Hormold/pubg

**Comment by javilobo8** [comment]
WOW... @Christian-nylander He copied my whole proyect xD https://github.com/PatyYe/pubg.op.gg-API-Wrapper

But yes.

Here is their API point: https://pubg.op.gg/api/users/${playerid}/ranked-stats?season=${season}&server=${server}&queue_size=${queuesize}&mode=${mode}

**Comment by dedarh** [comment]
@javilobo8 https://pubg.op.gg/api/matches/2U4GBNA0Ymn_16kHRoN5DuEGaAYyb3F3Bw1nZFP4tX0jpWrWna1AF2H1A7S4kZOq/deaths
https://pubg.op.gg/api/matches/2U4GBNA0Ymn_16kHRoN5DuEGaAYyb3F3Bw1nZFP4tX0jpWrWna1AF2H1A7S4kZOq

**Comment by javilobo8** [comment]
@Christian-nylander @dedarh @swtch  I have a solution, obtaining the same data. How can I contact you?

**Comment by dedarh** [comment]
@javilobo8  my email p.artem123@yandex.ru

**Comment by kevinreilly** [comment]
@javilobo8 @dedarh could you guys share your solution?

**Comment by dedarh** [comment]
@javilobo8 http://pubgtop.ru:81/api/opp_gg/nick/Jinheks (20 last game solo)
